src/bessel.jl

```diff
@@ -116,7 +116,7 @@ for afn in (:airyai, :airyaiprime, :airybi, :airybiprime,
         $afn(z::Complex64) = Complex64($afn(Complex128(z)))
     end
     if afn in (:airyaix, :airyaiprimex)
-        @eval $afn(x::Real) = x < 0 ? throw(DomainErrorNoArgs) : real($afn(complex(float(x))))
+        @eval $afn(x::Real) = x < 0 ? throw(DomainError(x, "`x` cannot be negative.")) : real($afn(complex(float(x))))
```

Comment:
Feel free to open an issue in `Base` if you feel strong about that. However, it should be clear since the notation follows what the docstring uses and for multiple argument domain errors it becomes necessary to know which variables are the ones that are throwing the domain error, so there is no way to work around that case.